python/python/tests/test_query.py

```diff
@@ -106,6 +106,12 @@ class TestModel(LanceModel):
     assert r0.float_field == 1.0
 
 
+def test_offset(table):
+    q = LanceVectorQueryBuilder(table, [0, 0], "vector").offset(1)
+    results = q.to_pandas()
+    assert len(results) == 2
```

Comment:
I'm confused. The table only has two rows. If you pass `offset(1)`, shouldn't there only be one result?

It's not clear to me this test is doing anything meaningful. You might need to update the `_execute_query(` method above.